backend/app/controllers/spree/admin/refunds_controller.rb

```diff
@@ -8,6 +8,22 @@ class RefundsController < ResourceController
 
       rescue_from Spree::Core::GatewayError, with: :spree_core_gateway_error
 
+      def create
```

Comment:
I don't really like them, but it might be better than what we're doing here? @jhawthorn I'll leave you as tie breaking vote.